core/src/main/res/values/dimens.xml

```diff
@@ -8,14 +8,16 @@
     <dimen name="text_size_navdrawer">16sp</dimen>
     <dimen name="text_size_medium">18sp</dimen>
     <dimen name="text_size_large">22sp</dimen>
+    <dimen name="text_size_semi_medium">18sp</dimen>
     <dimen name="thumbnail_length_itemlist">56dp</dimen>
     <dimen name="thumbnail_length_queue_item">56dp</dimen>
-    <dimen name="thumbnail_length_onlinefeedview">100dp</dimen>
+    <dimen name="newthumbnail_length_onlinefeedview">84dp</dimen>
```

Comment:
Please rename back to not have "new" in the name

Reply:
what should i name it? Please tell